.env.sample

```diff
@@ -1,3 +1,4 @@
 REACT_APP_CLIENT_ID=example
 REACT_APP_OKTA_ISSUER_URI="https://example-882474.okta.com"
-REACT_APP_API_URI=http://localhost:8005
\ No newline at end of file
+REACT_APP_API_URI=http://localhost:8005
```

Comment:
Has this been added to the correct .env file?

Reply:
Yes, we've got all the variables we need in the correct .env file.